lib/server/routes/buckets.js

```diff
@@ -948,28 +949,66 @@ BucketsRouter.prototype.getFile = function(req, res, next) {
       return next(new errors.NotFoundError('Bucket not found'));
     }
 
-    BucketEntry.findOne({
-      _id: req.params.file,
-      bucket: bucket._id
-    }).populate('frame').exec(function(err, entry) {
+    User.findOne({
```

Comment:
Nope, that is what I had thought. However the request is token authorized so user is not there. Also tried using populate on bucket, but this is also not working see: https://github.com/Storj/service-storage-models/issues/40